studio/LonaStudio/Documents/ImageDocument.swift

```diff
@@ -42,6 +42,11 @@ class ImageDocument: NSDocument {
 
     override func read(from url: URL, ofType typeName: String) throws {
         let data = try Data(contentsOf: url, options: NSData.ReadingOptions())
-        content = NSImage(data: data)
+		if url.pathExtension == "svg" {
```

Comment:
It looks like your indentation setting is set to tabs? Can you change it to 4 spaces?

![Screen Shot 2019-03-13 at 5 46 29 PM](https://user-images.githubusercontent.com/1198882/54323779-f5ff6d00-45b7-11e9-8503-cd2a50bc70e5.png)